app/backend/.env.sample

```
@@ -1 +1 @@
-DB_URL=mongodb://mongo:27017/task-manager
+DB_URL=mongo://mongo:27017/task-manager
```

это же название протокола, по которому нужно общаться с монгой. надо вернуть как было.

```suggestion
DB_URL=mongodb://mongo:27017/task-manager
```